packages/flex-plugin-e2e-tests/src/utils/pages/view/twilio-console.ts

```diff
@@ -0,0 +1,52 @@
+import { Page } from 'puppeteer';
+
+import { Base } from '../base';
+import { Cookie, Cookies } from '../../console-api';
+
+export class TwilioConsole extends Base {
+  assert = {};
+
+  private _loginForm = 'form[method="post"';
+
+  private readonly _baseUrl: string;
+
+  private readonly _flexBaseUrl: string;
+
+  constructor(page: Page, { flex, twilioConsole }: { flex: string; twilioConsole: string }) {
+    super(page);
+    this._baseUrl = twilioConsole;
+    this._flexBaseUrl = flex;
+  }
+
+  /**
+   * Logs user in through service-login
+   * @param cookies
+   * @param flexBaseUrl
+   * @param flexPath
+   * @param accountSid
+   */
+  async login(cookies: Cookies, flexPath: string, accountSid: string): Promise<void> {
+    const redirectUrl = this._flexBaseUrl.includes('localhost')
+      ? 'http://localhost:3000&localPort=3000'
```

Comment:
Make this localhost url a constant too - also what if want to run this on another port than 3000?